tests/integration/framework/process/daprd/daprd.go

```diff
@@ -45,16 +45,17 @@ type Daprd struct {
 	appHTTP  process.Interface
 	freeport *util.FreePort
 
-	appID            string
-	namespace        string
-	appProtocol      string
-	appPort          int
-	grpcPort         int
-	httpPort         int
-	internalGRPCPort int
-	publicPort       int
-	metricsPort      int
-	profilePort      int
+	appID                string
+	namespace            string
+	appProtocol          string
+	appPort              int
+	grpcPort             int
+	httpPort             int
+	internalGRPCPort     int
+	publicPort           int
+	metricsPort          int
+	profilePort          int
+	schedulerHostAddress string
```

Comment:
Make a pointer
```suggestion
	schedulerHostAddress *string
```

Reply:
I dont have those changes in this PR yet. Im waiting on the other PR to get merged to have this change

Reply:
I'll double check rq tho, and make sure that is reflected 👍🏻